src/views/Receive.vue

```diff
@@ -113,7 +114,8 @@ export default {
           NEAR: { name: 'NEAR', url: '' },
           SOL: { name: 'SOLANA', url: 'https://solfaucet.com/' },
           MATIC: { name: 'MATIC', url: 'https://faucet.matic.network/' },
-          ARBETH: { name: 'ARBETH', url: 'https://faucet.rinkeby.io/' }
+          ARBETH: { name: 'ARBETH', url: 'https://faucet.rinkeby.io/' },
+          FUSE: { name: 'FUSE', url: '' }
```

Comment:
If there is a faucet to get fuse, please add it here

Reply:
I will need to make a website and a faucet bot. I will try to implement a poc quickly.

Reply:
I'm going to place the discord invite while I build the bot and website.